### ISSUE TITLE
Selenium Error: This version of ChromeDriver only supports Chrome version 81

### ISSUE THREAD

**Issue by tatwell** [formulafolios/ann-arbor#2]
After following steps in README, when I run script I get error:

> selenium.common.exceptions.SessionNotCreatedException: Message: session not created: This version of ChromeDriver only supports Chrome version 81

To reproduce:

```
$ python app.py audit --crawl httpbin.org

[several lines of output skipped]

selenium.common.exceptions.SessionNotCreatedException: Message: session not created: This version of ChromeDriver only supports Chrome version 81
```

My version of ChromeDriver *is* version 81:

```
$ chromedriver --version
ChromeDriver 81.0.4044.69 (6813546031a4bc83f717a2ef7cd4ac6ec1199132-refs/branch-heads/4044@{#776})
```

¯\\\_(ツ)\_/¯

**Comment by tatwell** [comment]
This Stack Overflow answer looks promising:

- https://stackoverflow.com/a/61412036/9381758

I will test that out as soon as I have a chance.

**Comment by bennpham** [comment]
I'm curious what version of Chromium are you using?

I'm using Ubuntu on Windows with WSL2 (its a bit more finicky to setup because Windows) but I manage to run the following without issues

```
python3 app.py audit --crawl httpbin.org
```

If I run `chromium --version` and `chromedriver --version` the 2 versions should match. I got my chromium off snap via `snap install chromium`.

**Comment by tatwell** [comment]
The fix recommended in [this SO answer](https://stackoverflow.com/a/61412036) worked for me:

```
$ python app.py audit --crawl httpbin.org

[many lines removed]

aXe Site Audit Summary
----------------------
domain:         httpbin.org
pages:          1
violations:     5
\- errors:      5
\- warnings:    0

Top Templates by Violations:
forms: 5

Top Subtemplates by Violations:
forms/post: 5

created:        2020-09-24 18:51:53
runtime:        0:00:04.519313

Violations CSV: /home/klenwell/projects/ann-arbor/audits/httpbin-org/httpbin-org-site-all-violations.csv
```

Nice. :smiley:

**Comment by kateugenio** [comment]
Per request, I ran same command above. Results:

```
aXe Site Audit Summary
----------------------
domain:         httpbin.org
pages:          1
violations:     5
\- errors:      5
\- warnings:    0

Top Templates by Violations:
forms: 5

Top Subtemplates by Violations:
forms/post: 5

created:        2020-09-24 19:28:01
runtime:        0:00:06.504531

Violations CSV: /Users/keugenio/Documents/Python/ann-arbor/audits/httpbin-org/httpbin-org-site-all-violations.csv
```

**Comment by tatwell** [comment]
## Resolved
Fixed together with Issue #1 by adding `webdriver-manager` Python package.

I tested it with Ubuntu 18. Katherine tested it on Mac OS.
